packages/cli/src/util/oauth.ts

```diff
@@ -382,22 +400,54 @@ function canParseURL(url: string) {
   }
 }
 
-interface VercelAccessToken extends JWTPayload {
-  team_id?: string;
+/** @see https://datatracker.ietf.org/doc/html/rfc7662#section-2.2 */
+interface Token {
+  /**
+   * Integer timestamp, measured in the number of seconds
+   * since January 1 1970 UTC, indicating when this token will expire.
+   */
+  exp: number;
+  /** Whether or not the presented token is active. */
+  active: boolean;
+}
```

Comment:
If we stop checking the refresh token's `exp`, then I think we can remove this type, and simplify the return type of `inspectToken`. `inspectToken` could be as simple as calling `decodeJwt`. WDYT?